### ISSUE TITLE
[Bug]: Doesn't work any longer since 1.0.1.0

### ISSUE THREAD

**Issue by matternjei** [pnd280/complexity#56]
### Version

1.0.6.0

### Describe the bug

since the release of 1.0.1.0 it has never worked with arc browser. Even the latest version 1.0.6.0  doesn't work either. The only version that works with arc browser is 0.0.5.19

### Steps to reproduce

i am not able to reproduce since the extension doesn't work. 

### Bug confirmation

- [X] I am not able to reproduce the issue with the extension disabled
- [X] I have re-installed the extension and the issue persists

### Expected behavior

The various models are supposed to show with extra layer of features.

### Browser

Arc browser

**Comment by pnd280** [comment]
Hi @matternjei, could you give more details on what doesn't work?

1. What is the exact version of your Arc Browser and Chromium core? (`arc://settings/help`)
    ![image](https://github.com/user-attachments/assets/f1a1d41e-721f-4659-aabc-e9b7bddcfd37)

2. What is the behavior when you load `perplexity.ai`?
    - The extension doesn't load at all:
      - there is no changes to the UI (e.g. no model selector on the query box)
      - color scheme from custom theme doesnt apply
    - The extension does load the theme (blue, purple, yellow etc) but I can't use other features

Please reply to this thread with specified details, alongside a screenshot of the page and browser console (press F12 => "Console" tab). Thanks!
  
![image](https://github.com/user-attachments/assets/2a105d55-cb0e-4287-8f5e-de8acef92c86)

**Comment by matternjei** [comment]
The extension doesn't load at all and there is no change to the UI. 
![Screenshot 2024-12-24 054330](https://github.com/user-attachments/assets/a5fad79c-da3a-4cda-8106-91e470ce1d5a)
![perplexity](https://github.com/user-attachments/assets/7a7a17ed-e368-4aeb-a1d8-bf797437cc26)
![Screenshot 2024-12-24 054041](https://github.com/user-attachments/assets/ec4550c6-1bd6-47c5-a89a-3c459bb335f6)

**Comment by matternjei** [comment]
![console](https://github.com/user-attachments/assets/65fbba0a-1999-4935-963b-6d7fa59d91ea)

**Comment by pnd280** [comment]
I'll close this issue, please update to the latest version, and reopen this issue if it persists.

**Comment by matternjei** [comment]
Yes. I have tested it with latest version and its working fine. Thank you so much
